packages/frontend/src/utils/dashboardSavedFiltersOverride.test.ts

```diff
@@ -0,0 +1,340 @@
+import { ConditionalOperator, DashboardFilterRule } from '@lightdash/common';
+import { createOverrideDashboardSavedFiltersUrlSubParam } from './dashboardSavedFiltersOverride';
+
+describe('dashboardSavedFiltersOverride', () => {
+    it('should override a boolean saved filter', () => {
+        const originalBooleanFilter: DashboardFilterRule = {
+            id: 'e7df7c5a-1070-439a-8300-125fe5f9b1af',
+            target: {
+                fieldId: 'orders_is_completed',
+                tableName: 'orders',
+            },
+            values: [true],
+            label: 'Is Completed',
+            operator: ConditionalOperator.EQUALS,
+        };
+
+        // Just Value changed
+        expect(
+            createOverrideDashboardSavedFiltersUrlSubParam(
+                originalBooleanFilter,
+                {
+                    ...originalBooleanFilter,
+                    values: [false],
+                },
+            ),
+        ).toBe('orders_is_completed.equals:false');
+
+        // Just Operator changed
+        expect(
+            createOverrideDashboardSavedFiltersUrlSubParam(
+                originalBooleanFilter,
+                {
+                    ...originalBooleanFilter,
+                    operator: ConditionalOperator.NULL,
+                    values: [],
+                },
+            ),
+        ).toBe('orders_is_completed.isNull');
+
+        // Both Operator and value changed
+        expect(
+            createOverrideDashboardSavedFiltersUrlSubParam(
+                originalBooleanFilter,
+                {
+                    ...originalBooleanFilter,
+                    operator: ConditionalOperator.NOT_NULL,
+                    values: [false],
+                },
+            ),
+        ).toBe('orders_is_completed.notNull:false');
```

Comment:
Maybe we should make sure the boolean ones are symetrical? Like, you have 
`orders_is_completed.isNull` with no value, but `orders_is_completed.notNull:false`. 

Would it make more sense if these were either both valueless like `orders_is_completed.isNull` and `orders_is_completed.notNull`

OR

`orders_is_completed.isNull:true` and `orders_is_completed.notNull:true` ?

Reply:
> orders_is_completed.isNull

Agree with this approach! updating the code :)